tests/unit/amo/components/TestAddonReviewList.js

```diff
@@ -540,13 +546,11 @@ describe(__filename, () => {
           },
         ],
       };
-      dispatchAddon(addon);
-      const root = render();
+      const header = renderAddonHeader({ addon });
+      const h3 = header.find('.AddonReviewList-header-authors').render();
 
-      const h3 = root.find('.AddonReviewList-header-authors');
-
-      expect(h3.render().text()).toEqual('by Chantal, Leroy');
-      expect(h3.render().find('a')).toHaveLength(2);
+      expect(h3.text()).toEqual('by Chantal, Leroy');
+      expect(h3.find('a')).toHaveLength(2);
```

Comment:
Should the test also be checking that the `href` of the `a` tags are the expected values?